src/OneSignal.ts

```diff
@@ -362,6 +364,19 @@ export default class OneSignal {
     await OneSignal.context.promptsManager.internalShowNativePrompt();
   }
 
+  /**
+   * Shows delayed prompt
+   * @PublicApi
+   * @param type - type of prompt: "native" | "slidedown"
+   * @param timeDelaySeconds - number of seconds to delay the prompt by
+   * @Example
+   *  OneSignal.push(function() { OneSignal.showDelayedPrompt("native", 20); });
+   */
+  public static async showDelayedPrompt(type: DelayedPromptType, timeDelaySeconds: number): Promise<void> {
```

Comment:
I don't see lot of use in this `showDelayedPrompt` public method. It would be the same as them calling.
```javascript
OneSignal.push(function() {
   setTimeout(OneSignal.showNativePrompt, 20000);
});
```

I would prefer we remove this for this release, otherwise we will be stuck with this public API which will have to be supported for awhile.